### ISSUE TITLE
🥇 Win the SciPy 2024 - prefix.dev puzzle 🧩 

### ISSUE THREAD

**Issue by ruben-arts** [prefix-dev/pixi#1607]
**TL;DR:** Like (👍) this issue to enter in winning the jigsaw puzzle at the SciPy 2024 Tacoma WA.

We've created a jigsaw puzzle specific for the SciPy 2024 and together with the attendees we put it together. Now it is done we would like to give it away. We use this issue to keep track of the people who like to win it.

To enter the lottery like this post and we'll pick a winner from the list of likes. 

**Disclaimer:**
- You need to be **in person at the conference** to win, as we will hand it out at the end of the conference.

![paxton_scipy_puzzle_2024_finished](https://github.com/user-attachments/assets/895ff4d7-785a-4ccf-9110-b8fc3f9dc5c4)

Ps. don't forget to star the project 😉 

The puzzle was put together by at least 40 people and a group of 14 that did a big part of it. It was an awesome way to get to know you all!

**Comment by ruben-arts** [comment]
Congrats @QuLogic on winning the puzzle. And thanks everyone for liking the issue!

Hope to see you next year for the new puzzle!
